### PR TITLE
Migrate mongodb-express to vault

### PR DESCRIPTION
**Pull request by Nexus2k**
https://github.com/w3f/SecOps/issues/213

### CODE REVIEW

File: helmfile.d/config/mongo-express-common.yaml.gotmpl [hunk 1]
```diff
@@ -0,0 +1,18 @@
+environment: {{ .Environment.Name }}
+
+image:
+  tag: v0.1.0
```
Comment (ironoa) [inline helmfile.d/config/mongo-express-common.yaml.gotmpl:4]: I'd remove this, and let the chart values.yaml decide the image tag

File: helmfile.d/config/mongo-express-common.yaml.gotmpl [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+basicauth:
+{{ if eq .Environment.Name "production" }}
```
Comment (ironoa) [inline helmfile.d/config/mongo-express-common.yaml.gotmpl:7]: can we remove this "if" by having a default/common false, and a prod (in the prod file) true ?

File: helmfile.d/config/mongo-express-common.yaml.gotmpl (hunk 1 above)
Comment (Nexus2k) [reply helmfile.d/config/mongo-express-common.yaml.gotmpl:4]: ack

File: helmfile.d/config/mongo-express-common.yaml.gotmpl (hunk 2 above)
Comment (Nexus2k) [reply helmfile.d/config/mongo-express-common.yaml.gotmpl:7]: good point

**Review by ironoa — APPROVED** [review]

**Pull request merged**